src/app/assembly-bench/bench-list/bench-list.component.ts

```diff
@@ -1,25 +1,31 @@
-import { Component, OnInit, EventEmitter, Output } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { Pc } from '../pc.model';
+import { BenchService } from '../bench-list.service';
 
 @Component({
   selector: 'app-bench-list',
   templateUrl: './bench-list.component.html',
   styleUrls: ['./bench-list.component.css']
 })
 export class BenchListComponent implements OnInit {
-  @Output() pcWasSelected = new EventEmitter<Pc>();
-  assebledPcs: Pc[] = [
+  // Old just for reference @Output() pcWasSelected = new EventEmitter<Pc>();
+  assebledPcs: Pc[];
```

Comment:
It's always better not only to define a variable, but to assign it value (to remove the possibility of error).
Ex: `public assebledPcs: Pc[] = []`